web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -54,6 +54,23 @@ export class DataStoreService {
       );
   }
 
+    /**
+   * Returns an Observable that loads and emits the projects with the specified
+   * uuid.
+   *
+   */
+  loadAllProject$() {
+      return this.db
+        .collection('projects')
+        .snapshotChanges().pipe(
+          map(actions => actions.map(a => {
+            const data = a.payload.doc.data() as Project;
```

Comment:
`data`  isn't a `Project` until it's fed through the converter below. Instead, replace this with `as DocumentData` and remove the extra cast below.